### ISSUE TITLE
[Bug]:  laravel.log 48Gb

### ISSUE THREAD

**Issue by DCRalph** [coollabsio/coolify#3578]
### Error Message and Logs

Server fails to do almost anything because disk is full.  laravel.log in this dir /var/lib/docker/overlay2/db14921cf6decb89c5df51eca18453efac32e5f85b3ccdc365b43dc62ece08a1/diff/var/www/html/storage/logs is taking up 48gb after about a month of running with 2 apps.

### Steps to Reproduce

1. Install coolify
2. setup apps
3. wait 1 month

### Coolify Version

cant access dashboard

### Are you using Coolify Cloud?

No (self-hosted)

### Operating System and Version (self-hosted)

Ubuntu 22.04.4 LTS 

### Additional Information

The apps running were still accessible but the coolify dash was not

**Comment by DCRalph** [comment]
i delete the log file and updated to v4.0.0-beta.343

**Comment by ckarsan** [comment]
Yeah I was getting similar, I was wondering if there is there any automation or setting somewhere just to delete the logs, or limit the size / overwrite once a certain size.

**Comment by JakeKydd** [comment]
where can I find the log file?

**Comment by nicklascook** [comment]
I had the exact same issue. Fresh VPS, with only a next.js app running for around 1-2 months.
```
$ sudo df -h
...
overlay          75G  6.4G   66G   9% /var/lib/docker/overlay2/0408ea63d31930e7b12907ce15df1700765f0bdfbc8474382b2514bdd6a03878/merged
overlay          75G  6.4G   66G   9% /var/lib/docker/overlay2/c2c2364e0b48ea29e46b127c49b40bd733b4958525c8cf28e1650c95e4ecc7a6/merged
overlay          75G  6.4G   66G   9% /var/lib/docker/overlay2/93aa0a8135d4763b7c863f7912b98510c468b622d17e41e9003d9ce14e9d474b/merged
...
```

```
$ sudo du -sh /var/lib/docker/overlay2/c2c2364e0b48ea29e46b127c49b40bd733b4958525c8cf28e1650c95e4ecc7a6/merged/var/www/html/storage/logs/laravel.log
68G	/var/lib/docker/overlay2/c2c2364e0b48ea29e46b127c49b40bd733b4958525c8cf28e1650c95e4ecc7a6/merged/var/www/html/storage/logs/laravel.log
```

**Comment by DCRalph** [comment]
fix is easy as just delete all that crap

**Comment by cvpfus** [comment]
> fix is easy as just delete all that crap

how did you remove them?
